### PR TITLE
Standardize sample projects, Part 18

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: samples/snippets/csharp/VS_Snippets_CLR/conceptual.resources.locating/cs/assemblyinfo.cs [hunk 1]
```diff
@@ -1,40 +0,0 @@
-﻿using System.Reflection;
```
Comment (Youssef1313) [inline samples/snippets/csharp/VS_Snippets_CLR/conceptual.resources.locating/cs/assemblyinfo.cs:1]: As *assemblyinfo.cs* files are deleted, should we explicitly set [GenerateAssemblyInfo](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#generateassemblyinfo) to False in csproj?

Comment (NextTurn) [reply samples/snippets/csharp/VS_Snippets_CLR/conceptual.resources.locating/cs/assemblyinfo.cs:1]: > The documentation says the default value is True, so I'd expect them to be re-generated, is that correct?

Right. These deleted *AssemblyInfo.cs* files only contain auto-generated values.

**Review by BillWagner — APPROVED** [review]
Thanks again @NextTurn 

I'll :shipit: now.

**Comment by gewarren** [comment]
There's a build warning introduced here. Can someone fix it?

Invalid code snippet link: '~/samples/snippets/csharp/VS_Snippets_CLR/conceptual.resources.locating/cs/assemblyinfo.cs'.

**Comment by Youssef1313** [comment]
@gewarren I'll open a PR.

**Pull request merged**